js/components/contactDetails/contactDetails_controller.js

```diff
@@ -25,7 +26,8 @@ angular.module('contactsApp')
 		download : t('contacts', 'Download'),
 		delete : t('contacts', 'Delete'),
 		save : t('contacts', 'Save changes'),
-		addressBook : t('contacts', 'Address book')
+		addressBook : t('contacts', 'Address book'),
+		loading : t('contacts', 'Waiting for the contact list to load...')
```

Comment:
I’d say »Loading contacts« here, that doesn’t sound like we’re complaining about the slowness of our own app ;D

Reply:
Ahahha, good point!! 😂